cinn/runtime/buffer.cc

```diff
@@ -26,7 +26,7 @@ Shape::Shape(const Shape &other) : data_(new value_type[other.ndims()]), ndims_(
 void Shape::Resize(int ndim) {
   CHECK_GT(ndim, 0);
   ndims_ = ndim;
-  data_ = new value_type[ndim];
+  data_  = new value_type[ndim];
```

Comment:
Why do you think that the addition of a space character is needed here?

Reply:
I've not add space here. maybe the pre-commit changed it. I'll delete it. Thx.